src/Umbraco.Core/CodeAnnotations/UmbracoVolatileAttribute.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+
+namespace Umbraco.Core.CodeAnnotations
+{
+    /// <summary>
+    /// An empty attribute which is used as a marker for Roslyn Analyzers.
+    /// </summary>
+    /// <remarks>
+    /// Any class or method marked with this attribute will throw a Volatile error which can be suppressed
+    /// with <see cref="UmbracoSuppressVolatileAttribute"/>, this is intended for classes which are public
+    /// for testing purposes, but should not be accessed otherwise
+    /// </remarks>
```

Comment:
Again, just a small note about the relation to the Umbraco.Code

Reply:
I added a small note that the analyzer resides in the Umbraco.Code package and that it must be installed for the analyzer to have any effect.